Sources/SwiftCurrent/Extensions/DecoderExtensions.swift

```diff
@@ -0,0 +1,54 @@
+//  swiftlint:disable:this file_name
+//  DecoderExtensions.swift
+//  SwiftCurrent
+//
+//  Created by Tyler Thompson on 1/14/22.
+//  Copyright © 2022 WWT and Tyler Thompson. All rights reserved.
+//  swiftlint:disable file_types_order
```

Comment:
What caused the need for this disable?

Reply:
SwiftLint ;P

The issue was that DecodeWorkflow as a property wrapper was in this file, that has been moved.